appscale/tools/parse_args.py

```diff
@@ -272,8 +272,10 @@ def add_allowed_flags(self, function):
 
       group = self.parser.add_mutually_exclusive_group()
       group.add_argument('--appengine', type=int,
+        default=1,
         help="the number of application servers to use per app")
       group.add_argument('--autoscale', action='store_true',
+        default=True,
```

Comment:
I haven't seen `store_true` used with `default=True` before. Typically, the expectation with `store_true` is that `--flag` will make it true, and omitting it will make it false. One possibility would be to do `add_argument('--disable-autoscale', dest='autoscale', action='store_false')`. This would set args.autoscale to true unless `--disable-autoscale` was specified.